### ISSUE TITLE
Caldera Forms Processors exist but the fields for the processor do not show once saved.

### ISSUE THREAD

**Issue by lordofthegithub** [mecachisenros/caldera-forms-civicrm#94]
Happy New Year all,

I have a unique issue.

I have created few Caldera Forms with CiviCRM Processors few weeks ago. Today when I open those Caldera Forms, I can see the processors listed on the left, but when I click on that processor, the system doesn't show me any fields and mappings on the right.

So, I tried to create a new form from template; at the time of creation, I can see the fields in this new form, but upon saving, it behaves the same. It is not pulling the processor fields and mappings to variables or form fields. 

The only thing that has changed in the interim is that in WordPress I installed "Classic-Editor" plugin in order to see the CiviCRM block at the time of editing a WordPress Page/Post, so I can pull the custom profiles from CiviCRM, in case where Caldera Form doesn't provide processor for some functionality.

At this time it appears to be a WordPress ( version 5.0.2) & Caldera Forms ( version 1.7.5.1) issue; however, posting here, as this niche group knows more than just Caldera Forms & WordPress.... i.e. CiviCRM perspectives as well.

Many thanks in advance,

LOTG

**Comment by kcristiano** [comment]
@lordofthegithub I just tested a few Caldera Forms with CiviCRM.  WP 5.0.2 (With Classic Editor), CiviCRM 5.8.2, latest Caldera Forms and Caldera-Forms CiviCRM.    I can view and edit the forms and the processors.   

I am running the latest dev branch of Caldera-forms-civicrm, but I don't think that should make a difference in what you are describing.

Do you have a caching plugin installed?   I have found WP Caching plugins to be extremely problematic and have seen very strange behavior (even without CiviCRM).

**Comment by lordofthegithub** [comment]
@mwestergaard  and @kcristiano - Happy New Year and Thank You for prompt response. 
I do not have a caching plugin.. @mecachisenros indicated in a different issue that the caching plugins could be an issue.. so I removed the caching plugins long back.

I have latest versions of WordFence security, JetPack, Yoast SEO, and other task specific plugins that are not even remotely connected with caching functionality. 

One disclosure: my Caldera Forms is NOT a PRO or PAID version... could that be an issue ? 
I think I am closing in on 30 or 60 days of installing Caldera Forms on my site ? is the Free version a 30 day trial only ? 

This is Caldera Forms Processor specific issue, when creating I can see the fields and can map them to variables. However for subsequent EDIT or UPDATE of the processor fields just don't show up.. I can see the list of processors (Contact, Email, Phone, Case  etc) on the left, but when I click on the specific processor, the mappings just don't show on the right. All I see is a Remove Processor button.

I am guessing this could be licensing, but not entirely sure. Thoughts?

If this is intriguing to you, I'd be happy to connect offline and do a screen sharing for troubleshooting and discovery.

Many thanks again.

**Comment by lordofthegithub** [comment]
@mwestergaard - thank you again for prompt reply. I did check by changing the theme... still the same issue.. so it is not the theme issue .... and theme affects the Public website not the WordPress backend (in theory)... where you create and edit the Caldera Forms and processors.

**Comment by lordofthegithub** [comment]
Another Issue that I noticed is that that the target fields names in CiviCRM have changed and now they have a suffix, that I did not assign manually.. and this has happened to each custom field that that I've been using to populate via the Caldera Forms CiviCRM plugin by @mecachisenros …. not sure if there is a connection but it is certainly intriguing, and puzzling as I am new to all this.

![github 94 caldera forms processor target civicrm field names changed](https://user-images.githubusercontent.com/45275357/50632330-590bbe00-0efc-11e9-8ee2-2178845b586b.PNG)

**Comment by mecachisenros** [comment]
@lordofthegithub this has nothing to do with Caldera Forms (CF) free version or Pro, there's no time limit in using CF (free version) as it is free and open source.

Regarding the timestamp appended to the Dropdown name, that's always been true for any custom field with custom options (Checkbox, Dropdown, Radios, etc) created in Civi for a long time (if not always?) and it's not an issue, note aside, this plugin does not alter the custom options nor custom fields in any way.

Back to your issue, I suggest you disable all plugins (in a dev/stagging environment if possible) and test with only CF and Caldera Forms CiviCRM (CFC) enabled, and the Twenty Seventeen theme, to rule out any plugins or themes.

As I mentioned previously, I cannot reproduce the issue, so this must be something related to your server/hosting/environment.

**Comment by lordofthegithub** [comment]
@mecachisenros thank you for your reply. 
Looking into your suggestions... many thanks to this community for the help and suggestions... keep you all posted..

**Comment by mecachisenros** [comment]
@lordofthegithub any developments? Have you tried disabling all plugins and themes as suggested?

**Comment by lordofthegithub** [comment]
@mecachisenros thanks for the follow up... my production environment had various issues so I was unable to troubleshoot effectively for past 2 weeks. 
I don't have a environments cloned from production to test this.
and in dev or other environments that are not cloned from production, I am unable to replicate the issue.

I did try your suggestions today, with no success. 

I've just confirmed that with having ONLY Caldera Forms plugin active (rest of plugins deactivated), I am able to see the processors, the most basic ones. 
However, when I activate the related plugins, namely
a. CiviCRM (but just activating this plugin doesn't show the field processors in Caldera Forms, so I activate @mecachisenros plugin below)
b. Caldera Forms CiviCRM, even then no processors.

So at this point I have 3 plugins activated. But I am no longer able to see the field processors. 
Then I enable the Caldera Forms Anti Spam - and still no processors.
![github 94 - test 2019-01-28](https://user-images.githubusercontent.com/45275357/51871119-5a1ce900-230a-11e9-906c-92008089aa88.PNG)
I am unable to reason if this is a Word Press Theme issue. It more so appears to be some plugin issue between CiviCRM & the Caldera Forms CiviCRM , which is beyond my current level of expertise. 

I've opened a ticket with the Caldera Forms WP by purchasing a support package; and I am happy to let @mecachisenros know the ticket number with Caldera Forms WP; assuming you've got a good working relationship with them.

I am wondering if we can have a 3 way call to address this. may be @mecachisenros may be you take a look at this first (will send you credentials in an email ) and provide some guidance please.

Any breakthrough would be appreciated.

Thank you.

![github 94 - form processor view 2019-01-28](https://user-images.githubusercontent.com/45275357/51871464-40c86c80-230b-11e9-8d4c-6cda1a762c6a.PNG)

**Comment by lordofthegithub** [comment]
@mecachisenros - you're awesome. 

Thanks for signing in to the system and figuring out the issue in no time. The integration issue was due to incongruent data (field values - string instead of integer) originating from CiviCRM.

This did rescue me; however, I did lose my field mappings in the processors .. currently remapping to and testing. 
Thank to you @mecachisenros the issue is resolved.
